applications/nrf_desktop/configuration/common/hid_report_desc.h

```diff
@@ -23,28 +27,36 @@
 extern "C" {
 #endif
 
+/**
+ * @brief HID Reports
+ * @defgroup nrf_desktop_hid_reports HID Reports
+ * @{
+ */
+
+/** @brief Identification numbers of HID reports. */
 enum report_id {
-	REPORT_ID_RESERVED,
+	REPORT_ID_RESERVED,		/**< Reserved. */
```

Comment:
The comment /**< Reserved. */ should be before REPORT_ID_RESERVED.
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#enums

Same applies for all the parameters.